skietbaan_fe/src/components/Login.js

```diff
@@ -129,9 +135,19 @@ class Login extends Component {
 
     return (
       <div className="Page-content">
-      <div className = "welcome-header"><label className="welcome-label">Welcome back to skietbaan</label></div>
+      <div className="red-background">
+      <div className = "welcome-header">
+        <img src={require('../components/assets/header.png')} 
+        className="header-image"></img>
+      </div>
       <div className="header-container">
-      <label className = "header-label">Login</label>
+      <div className="centre-label">
+        <label className = "header-label">Login</label>
+        </div>
+      <img src={require('../components/assets/Back.png')} 
```

Comment:
Fix naming convention for images, previous image resource starts with a small letter

Reply:
fixed